lib/response-transforms.js

```diff
@@ -32,4 +32,8 @@ module.exports = {
       });
     }
   },
+  jsonify: res => {
```

Comment:
Just do this inline where we need it. Also why do we even need to do this? Does the server not respond with JSON? Have you set the correct "accept" header?